### PR TITLE
NAS-113263 / 13.0 / Update 13.0 stable branch

### PR DESCRIPTION
**Pull request by sonicaj**

### CODE REVIEW

File: src/freenas/usr/local/lib/middlewared_truenas/plugins/enclosure_/map.py [hunk 1]
```diff
@@ -40,6 +40,17 @@
             MappingSlot(0, 2, False),
         ]),
     ]),
+    ProductMapping(re.compile(r"(TRUE|FREE)NAS-MINI-3.0-X$"), [
```
Comment (yocalebo) [inline src/freenas/usr/local/lib/middlewared_truenas/plugins/enclosure_/map.py:43]: The `ProductMapping` section of this file seems to have many duplicated entries.

File: src/middlewared/middlewared/plugins/disk_/wipe.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 
 CHUNK = 1048576  # 1MB binary
+DIOCGMEDIASIZE = 0x40086481  # sys/sys/disk.h _IOR('d', 129, uint64_t)
```
Comment (yocalebo) [inline src/middlewared/middlewared/plugins/disk_/wipe.py:9]: This is not needed anymore and can be removed.

**Review by yocalebo — CHANGES_REQUESTED** [review]

**Review by yocalebo — APPROVED** [review]
SHIP IT!

**Comment by yocalebo** [comment]
I re-requested middleware team to take a look at this since it touches, quite literally, everything 😄

**Review by themylogin — APPROVED** [review]
Seems legit

**Review by Qubad786 — APPROVED** [review]
:shipit:

**Pull request merged**